apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -201,6 +201,12 @@ const Ticket = new GQLListSchema('Ticket', {
             defaultValue: false,
             isRequired: true,
         },
+        canReadByResident: {
+            schemaDoc: 'Determines if a resident in the mobile app can see the ticket created in crm',
+            type: Checkbox,
+            defaultValue: false,
```

Comment:
Are you shure?

Reply:
false to prevent old tickets from being displayed in the mobile app